dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -119,7 +124,7 @@ public class ResourcesController extends BaseController {
     @PostMapping(value = "/directory")
     @ApiException(CREATE_RESOURCE_ERROR)
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
-    public Result<Object> createDirectory(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
+    public Result<org.apache.dolphinscheduler.dao.entity.Resource> createDirectory(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```

Comment:
It's better to use a pojo for each api.

Reply:
Sorry. What i mean is we should do this in V2.

Reply:
```suggestion
    public Result<Resource> createDirectory(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```